src/client/pages/__tests__/create.test.tsx

```diff
@@ -1,15 +1,76 @@
+import '@testing-library/jest-dom';
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { render, screen } from '@testing-library/react';
 import Create from '../create';
 import { BrowserRouter as Router } from 'react-router-dom';
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(
-    <Router>
-      <Create />
-    </Router>,
-    div,
-  );
-  ReactDOM.unmountComponentAtNode(div);
+describe('<Create />', () => {
```

Comment:
```suggestion
describe('Create', () => {
```
(to be consistent with other tests)